pep_sphinx_extensions/tests/pep_processor/transform/test_pep_footer.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+
+from pep_sphinx_extensions.pep_processor.transforms import pep_footer
+
+
+def test_add_source_link():
+    out = pep_footer._add_source_link(Path("pep-0008.txt"))
+
+    assert (
+        str(out) == "<paragraph>Source: "
+        '<reference refuri="https://github.com/python/peps/blob/main/pep-0008.txt">'
+        "https://github.com/python/peps/blob/main/pep-0008.txt</reference>"
+        "</paragraph>"
```

Comment:
This all seems pretty tightly coupled with specific low-level details of the output formatting, so it seems like it'll likely need to be manually updated often if we make significant changes...but idk what to do better here.

Reply:
Yeah, we could always drop this test, the tested function isn't doing a lot:

```python
def _add_source_link(pep_source_path: Path) -> nodes.paragraph:
    """Add link to source text on VCS (GitHub)"""
    source_link = f"https://github.com/python/peps/blob/main/{pep_source_path.name}"
    link_node = nodes.reference("", source_link, refuri=source_link)
    return nodes.paragraph("", "Source: ", link_node)
```

Or `assert "https://github.com/python/peps/blob/main/pep-0008.txt" in out` instead.

Reply:
Hmm, good point. We could do either, or leave it as is; I leave it to your better judgement.

Reply:
Will go for the simpler assert: we want to make sure the link is good. (Plus let docutils do whatever it needs to do.)